dot/peerset/test_helpers.go

```diff
@@ -24,7 +25,7 @@ const (
 	peer2         = peer.ID("testPeer2")
 )
 
-func newTestPeerSet(t *testing.T, in, out uint32, bootNodes, reservedPeers []peer.ID, reservedOnly bool) *Handler {
+func newTestPeerSet(t *testing.T, ctx context.Context, in, out uint32, bootNodes, reservedPeers []peer.ID, reservedOnly bool) *Handler {
```

Comment:
Why not use `context.Background()` directly in there? I doubt we would have another use in tests right? Or maybe we would, not sure.

Reply:
yeah, I will change to use `context.Background()` directly!